docs/guide/dependencies.md

```diff
@@ -6,11 +6,9 @@ features.
 
 | Name | License | Author |
 | :--- | :--- | :--- |
-| [bessel](https://github.com/SheetJS/bessel) | Apache v2.0 | SheetJS |
+| [bessel](https://github.com/SheetJS/bessel) | Apache v2.0 | SheetJS | // in src/interpreter/plugin/3rdparty
 | [Chevrotain](https://github.com/SAP/chevrotain) | Apache v2.0 | SAP SE or an SAP affiliate company |
-| [core-js](https://github.com/zloirock/core-js) | The MIT License | Denis Pushkarev |
-| [jStat](https://github.com/jstat/jstat) | The MIT License | jStat |
-| [regenerator-runtime](https://github.com/facebook/regenerator/tree/master/packages/regenerator-runtime) | The MIT License | Facebook, Inc. |
+| [jStat](https://github.com/jstat/jstat) | The MIT License | jStat | // in src/interpreter/plugin/3rdparty
```

Comment:
The `// in src/interpreter/plugin/3rdparty` comments reduce the readability of the table.

I propose to add in another way, as a paragraph below the table:

> The bessel and jStat projects are distributed with the code repository at the path `src/interpreter/plugin/3rdparty` and bundled with the code package.

Reply:
done